### PR TITLE
Add appcat exoscale dbaas documentation

### PR DESCRIPTION
**Pull request by ZuGao**
We deployed the AppCat Exoscale Postgres service in test environment. It's time to update the documentatin.

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to/vshn-example/activate-exoscale-dbaas-appcat.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+vault login -method=oidc
+----
+
+. Setup API keys for exoscale cloud provider in case no other AppCat services are running in {zone}.
++
+.. Login to portal.exoscale.com
+.. Create a new organization using the naming scheme `VSHN AppCat ${CLUSTER_ID}`, for example `VSHN AppCat ch-gva-2-0`.
+.. Create an Exoscale support ticket to link the organization to the root organization.
+.. Once they're linked, in the new organization, create a restricted IAM API key using a name like `appcat-dbaas-cloudscale-ch-gva-2-0`.
+  The key should have following service permissions: `DBaaS - unrestricted`, `IAM - unrestricted`.
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/how-to/vshn-example/activate-exoscale-dbaas-appcat.adoc:27]: Maybe add a note that if buckets are also needed, that it should also have bucket access?

**Review by Kidswiss — APPROVED** [review]
Just one thing rest LGTM

File: docs/modules/ROOT/pages/how-to/vshn-example/activate-exoscale-dbaas-appcat.adoc (hunk 1 above)
Comment (ZuGao) [reply docs/modules/ROOT/pages/how-to/vshn-example/activate-exoscale-dbaas-appcat.adoc:27]: I added it in the warning clause, do you thing I should change the approach how I lay out the page?

Comment (Kidswiss) [reply docs/modules/ROOT/pages/how-to/vshn-example/activate-exoscale-dbaas-appcat.adoc:27]: Ah right, I was looking at it in the source, so I somehow skipped that as asciidoc code :D

**Review by Kidswiss — APPROVED** [review]

**Pull request merged**